puppet/modules/chassis/manifests/openssl.pp

```diff
@@ -0,0 +1,28 @@
+define chassis::openssl (
+) {
+	file { '/vagrant/certs':
```

Comment:
Not sure why this needs to provision files/dirs in the vagrant directory; is the idea that this allows easier access to install on the host?